### DISCUSSION TITLE
CVE-2023-24056 in Kong image

### DISCUSSION THREAD

**Discussion by kathpeony** [Kong/kong#10278] [Help] [answered]
Hi all,
during our black duck binary analysis [CVE-2023-24056](https://nvd.nist.gov/vuln/detail/CVE-2023-24056) popped up for Kong image.
I want to know if and how pkgconf is used in Kong. Is Kong gateway impacted by this vulnerability?

Otherwise, is it possible to upgrade the pkgconf version from 1.8.0-r1 to fixed version 1.8.1-r0

Thanks.

**Comment by fffonion** [comment, marked as answer]
@kathpeony Kong doesn't require `pkg-config` at run-time. It might be from the base image or being incorrectly
included in the release. Which image are you using?

**Reply by kathpeony** [reply]
@fffonion I am using the latest one v3.1.0.
Thanks, for the reply. So this means I can triage this finding and continue.

**Reply by fffonion** [reply]
@kathpeony Yes. We will also remove this deps in future version to avoid confusion. Please
do note the convient docker tag will also switch to `ubuntu` based instead of `alpine` based in another
2 minor releases.

**Reply by kathpeony** [reply]
Thanks for you help!
